README.md

```diff
@@ -92,13 +110,11 @@ Are you interested in contributing to kops? We the maintainers and community wou
 
 ## Office Hours
 
-Kops maintainers also have time (1 hour) set aside every other week to offer help and guidance to the
-community. This is the maintainer's **office hours**. During this time we might work with newcomers, help with PRs, and discuss new features. Anything goes.
+Kops maintainers set aside 1 hour every other week for **public** office hours for a video chat in which we strive to get to know the other developers either working on the project or interested in getting to know more about it. We tend to use this hour to discuss the current state of the kops project, to strategize about how to move it forward, ti discuss open and upcoming PRs, to demo, and to offer help and guidance to the community. Generally this time is focused on developers, although we will never turn a courteous participant away. Even if you've never actually installed kops, we're interested to have you stop by our office hours.
 
-We encourage you to reach out **beforehand** if you plan on attending. We have an [agenda](https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit) where we track notes from office hours.
+We encourage you to reach out **beforehand** if you plan on attending. Reaching out can be as simple as adding an item or your name to the [agenda](https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit) where we track notes from office hours.
 
-Office hours, on [Zoom](https://zoom.us/my/k8ssigaws) video
-conference are on Fridays at [5pm UTC/9 am US Pacific](http://www.worldtimebuddy.com/?pl=1&lid=100,5,8,12) every other week, on odd week numbers.
+Office hours, on [Zoom](https://zoom.us/my/k8ssigaws) video conference are on Fridays at [5pm UTC/12 noon ET/9 am US Pacific](http://www.worldtimebuddy.com/?pl=1&lid=100,5,8,12) every other week, on odd week numbered weeks.
```

Comment:
Grammar and spelling edit:

---

Office hours hosted on [Zoom](https://zoom.us/my/k8ssigaws) video conference, held on Fridays at [5 pm UTC/12 noon ET/9 am US Pacific](http://www.worldtimebuddy.com/?pl=1&lid=100,5,8,12) every other odd numbered week.